source/simulator/core.cc

```diff
@@ -1682,6 +1682,17 @@ namespace aspect
                                          T_preconditioner);
           solve_advection(AdvectionField::temperature());
 
+          if (
+            parameters.use_discontinuous_temperature_discretization
+            &&
+            parameters.use_limiter_for_discontinuous_temperature_solution
+          )
+            {
+              pcout << "Apply bound preserving limiter to the temperature advection field "
```

Comment:
I would prefer if we can move this message inside the `apply_limiter_to_dg_solutions` function (the same for the composition case). There is no functional reason except that this function is already really long and every line we can save in this function will keep the code easier to read.